ush/ioda/bufr2ioda/marine/b2i/bufr2ioda_insitu_surface_drifter.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import sys
+from b2iconverter.util import parse_arguments
+from b2iconverter.bufr2ioda_config import Bufr2iodaConfig
+from b2iconverter.bufr2ioda_converter import Bufr2ioda_Converter
+from drifter_ioda_variables import DrifterIODAVariables
+
+
+platform_description = 'Lagrangian drifter drogue profiles from dbuoy: temperature'
+
+
+class DrifterConfig(Bufr2iodaConfig):
+
+    def ioda_filename(self):
+        return f"{self.cycle_type}.t{self.hh}z.insitu_surface_drifter.{self.cycle_datetime}.nc4"
+
+
+if __name__ == '__main__':
+
+    script_name, config_file, log_file, test_file = parse_arguments()
+
+    bufr2ioda_config = DrifterConfig(
+        script_name,
+        config_file,
+        platform_description)
+
+    ioda_vars = DrifterIODAVariables()
+    ioda_vars.set_temperature_var_name("waterTemperature")
+    ioda_vars.set_temperature_error(0.02)
```

Comment:
Same comment as before, these error coefficient values are low. We might want to address this in a yaml config file as Andy suggestions.